Backfill events

elizabethengelman opened the pull request:
Backfill events for:
1. AAVE - flip, osm, median
1. BAL - flip, osm, median
1. ETH-B - flip
1. GUSD - flip
1. RENBTC - flip
1. UNI - flip, osm, median
1. UNIV2DAIETH - flip
1. YFI - flip, osm, median

Leaving this as a draft until https://github.com/makerdao/vdb-mcd-transformers/pull/418 is merged, and the invalid headers from 11234873-11234881 are removed.

elizabethengelman committed:
Add gusd, eth-b, yfi, bal, uni, renbtc, aave, univ2daieth to backfillEvents.toml

elizabethengelman committed:
Update backfillEvents formatting

elizabethengelman committed:
Add ENDING_BLOCK_NUMBER to backfillEvents startup script

elizabethengelman committed:
Add gusd, eth-b, yfi,bal, uni, renbtc, aave, univ2daieth to backfillEvents transformer exporter

paytonrules commented on dockerfiles/backfill_events/startup_script.sh:
```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+# The ENDING_BLOCK_NUMBER needs to be set to around the block number where VDB began tracking the collateral(s) events.
+# In other words, just after the block number where the new collateral(s) was added to exeucte.
```
exeucte / execute

rmulhol commented on dockerfiles/backfill_events/startup_script.sh:
```diff
@@ -1,10 +1,12 @@
     exit 1
 fi
-echo "Starting event backfill from block $ENDING_BLOCK_NUMBER"
+echo "Starting event backfill from block 11602562"
```
nbd and not introduced by this PR but maybe this should read "to block"

elizabethengelman committed:
Update backfill_events startup script

Manually set "to block" rather than setting it via an env var so that
was can more easily make changes.

Commit:
Merge branch 'prod' into backfill-events
